isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml

```diff
@@ -99,17 +99,6 @@ xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Applica
       process FIT files using the old keywords.  Also set TARGET=RYUGU in the event the user
       does not specify a target, and the target in the fit label is set to SKY.
     </change>
-    <change name="Kaitlyn Lee" date="2020-07-21">
-      Updated calibration method to match JAXAs calibration by:
-      1. moving linearity correction to after smear correction,
-      2. commenting out dark current and linearity correction,
-      3. changing an addition to subtraction in the g_bias calculation,
-      4. refactoring the smear correction to be before the main loop to speed up the program
-         and to be dependent on the bit-corrected input image DNs instead of the already corrected
-         output image DNs,
-      5. and removing `*g_sensitivity*g_texp` in the flat-field correction so that the output
-         aligns with l2b data.
-    </change>
```

Comment:
Why was this removed? Im sure there is a good reason, seems like it was moved to cal app?

Reply:
It was in the wrong file, hyb2onc2isis -vs- hyb2onccal